src/App.js

```diff
@@ -1,16 +1,40 @@
-import React from 'react';
+import React,{useState} from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import chatMessages from './data/messages.json'
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages)
+  console.log(`I'm in app ${entries}`)
+  const toggleLikes = (id) => {
+    setEntries(prevEntries => {
+      const updatedEntries = prevEntries.map(entry => {
+        return entry.id === id ? {...entry, liked: !entry.liked} : entry
+      })
+      return updatedEntries
+    })
+  }
+  const totalLikes = () => {
+    let total = 0;
+    for (let entry of entries) {
+      if (entry.liked){
+        total += 1;
+      }
+    }
+
+    return total;
+  };
```

Comment:
This is an optimal approach to calculating the number of likes, great job 😁

This is a great place to display the total likes